### DISCUSSION TITLE
[Proposal] Create Blog Sig

### DISCUSSION THREAD

**Discussion by longdafeng** [oceanbase/oceanbase#932] [General]
Hi, all TOC members, 

In order to provide more and more convient channel for developer to provide more and more experience or case documents to Community, we can to setup one Blog Sig. It will leading to setup one repo named blog,  at the same time,  all document of open.oceanbase.com/blog can be stored in this repo,  so every user can touch these blog by this way. 

Suggest Member:
fionaduan PMC(Chairman)
longdafeng PMC
caifeizhi PMC
sulang1 committer
askdb committer

**Comment by caifeizhi** [comment]
Should we distinguish between Chinese blogs and English blogs，blog-cn and blog-en?

**Reply by longdafeng** [reply]
In my opinion,  we can do as what we do for our documents, there are two directory in the repository, one is for Chinese, the other is English.

**Comment by LINxiansheng** [comment]
Agree

**Comment by longdafeng** [comment]
Since most of us are agree with this proposal. 

Let's vote for this propsal. 

+1 
0
-1

**Comment by longdafeng** [comment]
+1 (binding)

**Comment by xiangjiang-wn** [comment]
+1 (binding)

**Comment by Chuanhui** [comment]
+1 (binding)

**Comment by longdafeng** [comment]
Right now, all TOC members have voted this proposal, the result is as following:
+1 3
0 0
-1 0

So this proposal has been past according to this vote. 
In the next week, this proposal will upload to our offical website.
